### ISSUE TITLE
please check logs: qball3/hippostore rsync failure

### ISSUE THREAD

**Issue by stowler** [CVNRneuroimaging/infrastructure#130]
Hi Rob and Keith @rrmm @kmcgregor123456 ,

Failed rsync this morning: macula to qball3:/data/birc. Started xfer around 7:26 and ended by 7:58 it was hung. 

Ruled-out macula: it wan't doing anything else at the time. No errors in its logs.

Please check zabbix and qball3/hippostore logs.

Thanks,
Stephen

**Comment by rrmm** [comment]
hippostore seemed to have a load spike around that time.  I'm not sure what caused it.  Might be a cronjob; those sometimes run around that time.

**Comment by kmcgregor123456** [comment]
Is it still a problem?

**Comment by stowler** [comment]
No. Re-launched and completed macula->qball3:/data/birc ~60 GB rsync between 9am and 9:30. 

But FYI pano choked hard on whatever was happening with hippostore: after I found that I had two frozen vims (via ssh, outside of tmux), and tmux was hung. -ST

**Comment by stowler** [comment]
...so I rebooted pano. -ST

**Comment by stowler** [comment]
...and netuser home directories no longer mount after rebooting pano. Please fix. Thanks.

**Comment by kmcgregor123456** [comment]
netusers fixed. fstab had extra s in mount entry
